widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadLayout.kt

```diff
@@ -69,6 +69,10 @@ class ChatHeadLayout @JvmOverloads constructor(
         chatHeadView.showQueueing()
     }
 
+    override fun showScreenSharing() {
+        chatHeadView.showScreenSharing()
+    }
```

Comment:
Sorry, don't understand what it means - what exactly it will show? I thought we should navigate to the End Screen Sharing screen. 🤔

Reply:
Ah, I think I'm confused with the name. So, it doesn't show a screen but sets up the bubble to a 'screen sharing' look, right?